### PR TITLE
Graphical Update

### PR DESCRIPTION
**Pull request by int5749**
- added Iconset
- added rule
- fixed broken link in ToC
- updated itemal and items
- updated shortcuts
- added required files

- hope all is working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -328,141 +331,151 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
...
+            Text item=LandroidTotalBladeTime
         }
-        Setpoint item=LandroidRainDelay minValue=0 maxValue=750 step=30
-    }
-    Frame label="Statistic" {
-        Text item=LandroidTotalTime
-        Text item=LandroidTotalDistance label="Total Distance [%.2f km]"
-        Text item=LandroidTotalBladeTime
-    }
-    Frame label="Orientation" {
```
Comment (nibi79) [inline README.md:453]: 'Orientation' is missing

File: README.md [hunk 2]
```diff
@@ -245,9 +246,11 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
 
 ### .items
 ```
+String Shaun							"Shaun [%s]"
```
Comment (nibi79) [inline README.md:249]: ? I think we don't need this

File: README.md [hunk 3]
```diff
@@ -328,141 +331,151 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
...
+		Shaun.postUpdate(transform("MAP", "landroid_error_de.map", LandroidErrorCode.state.toString))
+	} else {
+		Shaun.postUpdate(transform("MAP", "landroid_status_de.map", LandroidStatusCode.state.toString))
+	}
+end
 ```
 
+Place the following *.map to your ..\conf\transform
+
+1. [landroid_error_de.map](https://github.com/nibi79/worxlandroid/doc/landroid_error_de.map)
```
Comment (nibi79) [inline README.md:466]: These files are not in your pull request

File: README.md (hunk 2 above)
Comment (int5749) [reply README.md:249]: We do, to have current rights away in main screen. So you can see whether Shaun is home, mowing or any error even without moving into next menu. Even i'ts not related to binding, it's unsing this to update and as well a good example for a working rule, which is required for this.

File: README.md (hunk 3 above)
Comment (int5749) [reply README.md:466]: Hmm, they should but not in images but in docs. Will check this

File: README.md (hunk 1 above)
Comment (int5749) [reply README.md:453]: OK, had this for testing under "move" in my sitemap and not migrated back :-(

This is now fixed.

File: README.md (hunk 3 above)
Comment (int5749) [reply README.md:472]: They are in my rep under docs, probabaly I don't have rights to upload those. As these are not images, how about a new folder "files" for this? Or where shall I place them?

Comment (int5749) [reply README.md:472]: This would leave "images" for Screenshots and "files" for all which is required to use this configuration.

**Pull request closed without merging**